stake-pool/cli/src/main.rs

```diff
@@ -1928,6 +1958,12 @@ fn main() {
                     .takes_value(true)
                     .help("Stake pool reserve keypair [default: new keypair]"),
             )
+            .arg(
+                Arg::with_name("yolo")
```

Comment:
Might want to go with a more specific flag name.  You never know when you'll need to yolo something else in the future :wink:

Reply:
hehe, do you suggest something like `--yolo-fees` then?

Reply:
How about `--unsafe-fees`? I'd hope "unsafe" is enough to make people scared?

Reply:
That probably makes more sense, thanks!